cmd/libs/go2idl/make-client/input/types.go

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
If you decide to proceed with my suggestion of using:
// +genclient=true
comment, please remove this file.